zoo-css-challenge/style.css

```
@@ -64,6 +65,11 @@ button {
   background-color: #372772;
   border: 2px solid black;
   color: #fff;
+  font-size: 125%;
+}
+
+button hover {
```

```suggestion
button:focus,
button:hover {
```
The CSS selector for hover is `:hover`, so we should change it to `button:hover`. We should also add `:focus` so that when keyboard users use the `Tab` key to move to the button it also shows a yellow background